### PR TITLE
Use rotate_180 rather than right_roll in dSight portrait mode.

### PR DESCRIPTION
**Pull request by DuFF14**
right_roll property is ignored. Use rotate_180 instead.

### CODE REVIEW

File: apps/displays/Sensics_dSight_portrait_2inputs.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
       {
         "center_proj_x": 0.5,
         "center_proj_y": 0.5,
-        "rotate_180": 0
+        "rotate_180": 180
```
Comment (rpavlik) [inline apps/displays/Sensics_dSight_portrait_2inputs.json]: should be 1 or true

File: apps/displays/Sensics_dSight_portrait_1input_sbs.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
       {
         "center_proj_x": 0.5,
         "center_proj_y": 0.5,
-        "rotate_180": 0
+        "rotate_180": 180
```
Comment (rpavlik) [inline apps/displays/Sensics_dSight_portrait_1input_sbs.json]: this is a bool masquerading as an int - don't recall exactly but pretty sure either 1 or true will do what you want. I know 1 will do what you want. I suspect 180 will not.

**Comment by DuFF14** [comment]
Looks like 1 and 180 do the same thing. I can change it to 1 though if that's clearer. Still waiting to confirm the dsight configurations are correct.

**Comment by rpavlik** [comment]
please do - then it matches the schema

**Pull request merged**